packages/connect/src/createPolkadotJsScClient/createPolkadotJsScClient.ts

```diff
@@ -22,12 +22,22 @@ export interface PolkadotJsScClient {
 
 type ResponseCallback = (response: string | Error) => void
 
+const subscriptionUnsubscritionMethods = new Map<string, string>([
+  ["author_submitAndWatchExtrinsic", "author_unwatchExtrinsic"],
+  ["chain_subscribeAllHeads", "chain_unsubscribeAllHeads"],
+  ["chain_subscribeFinalizedHeads", "chain_unsubscribeFinalizedHeads"],
+  ["chain_subscribeNewHeads", "chain_unsubscribeNewHeads"],
+  ["state_subscribeRuntimeVersion", "state_unsubscribeRuntimeVersion"],
+  ["state_subscribeStorage", "state_unsubscribeStorage"],
+  ["transaction_unstable_submitAndWatch", "transaction_unstable_unwatch"],
```

Comment:
All the methods of the old API are there, however `transaction_unstable_submitAndWatch` belongs to the new API.
We should either add all methods of both the old and new API (keeping in mind that the new API might change), or just the old API. I'd go for just the old API at the moment, leaving an explication as comment.